api/s2n.h

```diff
@@ -110,6 +110,15 @@ extern int s2n_config_set_extension_data(struct s2n_config *config, s2n_tls_exte
 extern int s2n_config_send_max_fragment_length(struct s2n_config *config, s2n_max_frag_len mfl_code);
 extern int s2n_config_accept_max_fragment_length(struct s2n_config *config);
 
+extern int s2n_config_set_session_state_lifetime(struct s2n_config *config, uint64_t lifetime_in_nanos);
+
+extern int s2n_config_disable_session_tickets(struct s2n_config *config);
```

Comment:
so if i get this right, existing implementations using s2n will need to explicitly call disable_session_tickets to maintain current behavior.  that seems backwards.

why would we default this to on?   also, there is no api to re-enable it.  should this instead be:

```
s2n_config_set_session_tickets_onoff(struct s2n_config *config, uint8_t enabled);
```

Reply:
Ahhh. Will change it to s2n_config_set_session_tickets_onoff.